aten/src/ATen/native/native_functions.yaml

```diff
@@ -2990,6 +2990,26 @@
     CPU: dense_to_sparse
     CUDA: dense_to_sparse
 
+- func: quantize_linear(Tensor self, float scale, int zero_point) -> Tensor
+  matches_jit_signature: True
+  variants: function, method
+  requires_tensor: True
```

Comment:
@jerryzh168 @gchanan - why do we need/have requires_tensor: True here?

Reply:
This is going to be removed in next pr.

Reply:
Added when I was trying to make the python API work since it was complaining something related to Variable.